OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,66 @@
+    <projectUrl>https://opentelemetry.io/</projectUrl>
+    <tags>Observability OpenTelemetry Monitoring Telemetry Tracing Auto Automatic Instrumentation</tags>
+    <repository type="git" url="https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation" />
+    <dependencies>
+      <group targetFramework="netcoreapp3.1">
+        <dependency id="OpenTelemetry" version="1.3.1" />
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" version="1.3.0-rc.2" />
+      </group>
+      <group targetFramework=".NETFramework4.6.2">
+        <dependency id="OpenTelemetry" version="1.3.1" />
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNet" version="1.0.0-rc9.5" />
+      </group>
+    </dependencies>
```

Comment:
Could you please elaborate on why this is needed? I thought that `dotnet pack` handles such stuff based on `csproj` content 🤷

Reply:
As explained in the problem description, want to keep few dependencies out of the Nuget package. In this way we may not spam the customer application and we will have only OpenTelemetry related assemblies.

Reply:
Also, if we use nuspec it won't look for dependencies list from csproj file.

Reply:
> As explained in the problem description, 

You are right. My brain "has not connected the dots" 😆